tests/analysis/Decompilation/dynamic_native/array.c

```diff
@@ -0,0 +1,27 @@
+#include <stdint.h>
+
+struct __attribute__((packed)) X {
+  uint16_t a[10];
+  uint64_t b;
+};
+
+uint16_t __attribute__((noinline)) getA(struct X *x) {
```

Comment:
Usually attributes go at the end of a *declaration*. Doesn't this give a warning?
Also, you're using `noinline` basically everywhere. I'd go for `-fno-inline` and be happy.

Reply:
How do I add this flag to the qa pipeline?

Reply:
1. Fyi this does not generate a warning, neither with gcc nor with clang. Output example:

```c
 →  gcc -Wall -Wextra -Wpedantic array.c        
array.c: In function ‘main’:
array.c:21:40: warning: unused parameter ‘argc’ [-Wunused-parameter]
   21 | int __attribute__((noinline)) main(int argc, char **argv) {
      |                                    ~~~~^~~~
```

2. Added `-fno-inline` as suggested